src/test/store/receive-profile.test.js

```diff
@@ -209,6 +210,27 @@ describe('actions/receive-profile', function() {
       ]);
     });
 
+    it('will show screenshot if the profile has screenshot data', function() {
+      const store = blankStore();
+      let profile = _getSimpleProfile();
+
+      store.dispatch(viewProfile(profile));
+      expect(getHumanReadableTracks(store.getState())).toEqual([
+        'show [process]',
+        '  - show [thread Empty] SELECTED',
+      ]);
+
+      profile = getScreenshotTrackProfile();
+
+      store.dispatch(viewProfile(profile));
+      expect(getHumanReadableTracks(store.getState())).toEqual([
+        'show [process]',
+        '  - show [thread Empty] SELECTED',
+        'show [screenshots]',
+        'show [screenshots]',
+      ]);
+    });
+
```

Comment:
@canova, I don't know if it should be done like this. Give me some suggestions here.